pokedex-with-bit/tests/test_pokedex.py

```diff
@@ -44,17 +44,13 @@ def get_flow():
     return f
 
 
-@pytest.fixture
-def object_image_paths():
-    return ['tests/test-data/1.png', 'tests/test-data/2.png', 'tests/test-data/3.png']
-
-
-def test_query(tmpdir, object_image_paths):
+def test_query(tmpdir):
     config(tmpdir)
     index_documents()
     f = get_flow()
     with f:
-        for object_image_path in object_image_paths:
+        from glob import iglob
+        for object_image_path in iglob(os.environ['JINA_DATA']):
             output = get_results(object_image_path)
             matches = output['search']['docs'][0]['matches']
```

Comment:
can we add a check that for every match `uri` is not empty?

Reply:
Done

Reply:
can u check if this exists for all of the matches and not only for the first one?